lib/join/joinv1/messages.go

```diff
@@ -267,6 +267,9 @@ func botParamsFromMessage(msg *messages.BotParams) *joinv1.BotParams {
 }
 
 func publicKeysToMessage(req *joinv1.PublicKeys) messages.PublicKeys {
+	if req == nil {
+		return messages.PublicKeys{}
+	}
```

Comment:
Should we be adding this nil check everywhere else too?
Or alternatively, should we replace the direct field access with the protobuf getter methods, so that the message is always checked for nil before accessing?

Reply:
switched to using the protobuf getters everywhere